include/oneapi/dpl/internal/random_impl/uniform_real_distribution.h

```diff
@@ -128,210 +128,148 @@ class uniform_real_distribution
     // Distribution parameters
     scalar_type a_;
     scalar_type b_;
+    template <class _Engine>
+    inline scalar_type
+    scale_and_displace(const scalar_type& __in, const param_type& __params, _Engine& __engine)
```

Comment:
Shouldn't `__engine` have also `const` qualifier?

Reply:
added

Reply:
will be moved to a single commit as it causes problems on windows which are needed to be investigated